src/parse.rs

```diff
@@ -30,6 +31,8 @@ use crate::{sentences::*, Error, SentenceType};
 pub const SENTENCE_MAX_LEN: usize = 102;
 
 /// A known and parsable Nmea sentence type.
+#[derive(Display)]
+#[display("${talker_id},{message_id},{data}*{checksum}")]
```

Comment:
The prefix for the sentence start is different for `AIS` messages!

- [ ] Support multiple prefixes